Crawler/Analyzers/UtilAnalyzers/PunctuationAnalyzer.cs

```diff
@@ -80,5 +91,45 @@ private List<int> WordsCountBetweenPunctuations(List<Token> tokens)
 
             return wordsCount;
         }
+
+        public double CalculatePassiveVoiceSentencesPercentage(List<PosTagToken> posTagTokens)
+        {
+	        var toBeForms = ExtractWordsFromFile(config.Value.ToBeFormsFile);
+	        var passiveVoiceSentencesCount = 0;
+	        var indexOfNextSentenceEnd = posTagTokens.FindIndex(token => token.ExtendedType == ".");
+            var totalSentencesCount = posTagTokens.Count(token => token.ExtendedType == ".");
+
+	        if (totalSentencesCount == 0)
+	        {
+		        totalSentencesCount = 1;
+	        }
+
+	        while (indexOfNextSentenceEnd != -1)
+	        {
+		        var sentence = posTagTokens.Take(indexOfNextSentenceEnd).ToList();
+
+		        var indexOfToBeForm = sentence.FindIndex(token => toBeForms.Contains(token.Value));
```

Comment:
what about lower case?

Reply:
What about wasn't? and every other not

Reply:
I will add the uppercase and the *n't versions as well.